adafruit_rfm9x.py

```diff
@@ -494,6 +498,10 @@ def tx_power(self):
         high power devices (RFM95/96/97/98, high_power=True) or -1 to 14 for low
         power devices. Only integer power levels are actually set (i.e. 12.5
         will result in a value of 12 dBm).
+        The actual maximum setting for high_power=True is 20dBm but for values > 20
+        the PA_BOOST will be enabled  resultiung in ad additonla gain of 3dBm.
```

Comment:
typos

Reply:
sigh...My fingers are usually way ahead of my brain.

Reply:
Yup, happens to me too. No worries. :-)